helper_crates/vtable/src/vrc.rs

```diff
@@ -340,32 +331,19 @@ impl<VTable: VTableMetaDropInPlace + 'static, X: HasStaticVTable<VTable> + 'stat
     }
 }
 
-/// Safety: The data VRc manages is held by `VRcInner`, which maintains its address when the VRc
-/// is moved.
-unsafe impl<VTable: VTableMetaDropInPlace + 'static, X> owning_ref::StableAddress
```

Comment:
I wonder if we still need to depends on the stable address trait crate and keep implemented these traits
otherwise it is a semver breaking change.

Reply:
You're right - I'll fix that :)